src/registry.rs

```diff
@@ -166,7 +163,7 @@ impl Registry {
 /// System wide actors registry
 ///
 /// System registry serves same purpose as [Registry](struct.Registry.html),
-/// except it is shared across all arbiters.
+/// except that it is shared across all arbiters.
```

Comment:
Is `that` needed here actually? I thought it is either `except` or `except for`

Reply:
Hm... I'm not sure what is right here because I rarely hear `except that`, regardless thank you